src/weatherreport/Makefile

```diff
@@ -0,0 +1,27 @@
+.PHONY: rel deps test
```

Comment:
As previously stated, if this is tied into the top-level Makefile / rebar.config.script, we should drop this file.

Reply:
Removed